config/P/config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "snowman-api-enabled": false,
+    "coreth-admin-api-enabled": false,
+    "coreth-admin-api-dir": "",
+    "state-sync-enabled": false,
```

Comment:
Why not we are enabling this, it is enabled in c-chain ?

Reply:
P-chain is the one who is going to create blocks and validate so we should do state sync don't we ? Maybe read more from here to make a informative decision https://docs.avax.network/nodes/configure/chain-config-flags#state-sync-enabled-boolean

Reply:
The thing is, it starts syncing with the testnet. And unless until it is completely synced, it wont allow me to query through the node

Reply:
Okay then keep as it is.